### ISSUE TITLE
[Feature Request] Yakuake

### ISSUE THREAD

**Issue by re1** [ishovkun/SierraBreeze#13]
Hey there,
First things first: This theme is amazing and I love it!
Now to my question. I would love to see a matching theme for the drop down Shell [Yakuake](https://github.com/KDE/yakuake).
Here is a screenshot of my current shell setups for reference:
![desktop](https://user-images.githubusercontent.com/10999017/35061075-08dff642-fbb8-11e7-9601-875c6b26d138.png)
I think that they fit "okay" but not quite.. Sadly I am a bit out of time right now and will not be able to do it myself.

**Comment by andreyorst** [comment]
Actually, [Matija Skala](https://github.com/matijaskala) already made [breezemite-yakuake
](https://github.com/matijaskala/breezemite-yakuake) theme.

**Comment by re1** [comment]
I have already tried out @matijaskala's version [1] but ended up removing it because I could not make it fit my needs in terms of icons size and colors.

[1] November 17

**Comment by matijaskala** [comment]
@re1 I did [a quick edit to the theme in your screenshot](https://github.com/ishovkun/SierraBreeze/files/1843598/yakuake-breezemite_perfect_dark-2.0.tar.gz).

**Comment by re1** [comment]
First, thank you so much for the effort! I liked your theme already and you really made it fit for my last environment. Sadly I switched to Arc KDE and also managed to decrease the button size so it does not fit anymore. Of course this is not the reason I am writing you (I do not want to waste your valuable time) it is the large shadow below the shell which bothers me.

![image](https://user-images.githubusercontent.com/10999017/37856731-fc386154-2eed-11e8-8ceb-d1f55f90ba15.png)

Note that I did not recompiled Yakuake with the changes suggested in the [breezemite-yakuake](https://github.com/matijaskala/breezemite-yakuake)'s README due to the extra effort and because I usually avoid making changes during compilation.

@matijaskala, if this version becomes available (maybe as a new branch) I will close this feature request and refer to your Yakuake skin instead. Thank you again and leave me a note if I can help you out somewhere in return!

PS: The corners were just a little more rounded :wink:
